test/integration/base_integration_test.cc

```diff
@@ -73,14 +79,22 @@ BaseIntegrationTest::BaseIntegrationTest(const InstanceConstSharedPtrFn& upstrea
 #endif
 }
 
+BaseIntegrationTest::BaseIntegrationTest(const InstanceConstSharedPtrFn& upstream_address_fn,
+                                         Network::Address::IpVersion version,
+                                         const std::string& config)
+    : BaseIntegrationTest(upstream_address_fn, version, configToBootstrap(config)) {}
+
+const BaseIntegrationTest::InstanceConstSharedPtrFn
+BaseIntegrationTest::defaultAddressFunction(Network::Address::IpVersion version) {
+  return [version](int) {
```

Comment:
Thoughts on either removing the int arg (since unused), or instead of capturing the version pass that as the lambda's argument?

Reply:
unused here but I think used elsewhere - I'd prefer to not change the signature or the default function (which I pulled out of the .h file) in this PR - I think it'd get messy :-(